test/file-test.js

```diff
@@ -8,9 +8,10 @@ describe('File', function () {
     it('looks up the content type', function () {
       var file = new File(5, {}, {});
 
-      assume(file.contentType('hello.js')).equals('application/javascript');
-      assume(file.contentType('hello.html')).equals('text/html');
-      assume(file.contentType('hello.dfadfasdf')).equals('application/octet-stream');
+      assume(file.contentDetect('hello.js').type).equals('application/javascript');
+      assume(file.contentDetect('hello.html').type).equals('text/html');
```

Comment:
Do we need to specify the charset?  e.g. should this be `text/html; charset=utf-8`?

Reply:
@msluther in terms of uploading it to s3 I don't think so. unless we want to forcibly append that but it hasn't been an issue thus far.